src/ccl/ccl.component.ts

```diff
@@ -3,20 +3,19 @@
  * App Component
  */
 import { Component, ViewEncapsulation } from '@angular/core';
+import { Sidebar } from './sidebar/sidebar.component';
+import { Charts } from './charts/charts-container.component';
+
 
-/*
- * App Component
- * Top Level Component
- */
 @Component({
   selector: 'ccl',
+  directives: [Sidebar, Charts],
   encapsulation: ViewEncapsulation.None,
   styleUrls: [
-  '../assets/css/main.css',
+    './ccl.style.css',
   ],
   templateUrl: './ccl.template.html'
```

Comment:
Same here, should be `ccl.component.html`. If there are other files named inconsistently in the project, they should be updated.